docs/guides/update-deprecated-resources.md

```diff
@@ -160,6 +156,10 @@ resource "aiven_organization_group_project" "example" {
       }
       ```
 
+      ~> **Important**
+      Do not use the `aiven_project_user` or  `aiven_organization_group_project` resource with `aiven_organization_permission`.
```

Comment:
Can we make it clear that this might lead to unexpected consequences? I'm not sure of the best wording, but something like this might work: These all control permissions at the organization level and may produce unexpected results when used together.

Reply:
What about: 

> Do not use the `aiven_project_user` or  `aiven_organization_group_project` resource with `aiven_organization_permission`. All of these resources manage organization-level permissions and using them together can cause conflicts and unexpected behavior.

Reply:
Yeah, that is much better, thank you :)